salt/modules/rabbitmqadmin.py

```diff
@@ -0,0 +1,195 @@
+# -*- coding: utf-8 -*-
+'''
+Module to provide RabbitMQAdmin compatibility to Salt.
+Todo: .
+'''
+from __future__ import absolute_import
+
+# Import salt libs
+import salt.utils
+
+# Import python libs
+import logging
+import json
+
+from rabbitmq import _format_response
+
+log = logging.getLogger(__name__)
+
+
+def translate_boolean(v):
+    if v:
+        return 'true'
+    else:
+        return 'false'
+
+
+def __virtual__():
+    '''
+    Only load if RabbitMQ rabbitmqcadmin is installed.
+    '''
+    return salt.utils.which('rabbitmqadmin') is not None
+
+
+def declare_queue(name, vhost, durable, auto_delete, runas=None):
+    '''
+    Adds a queue via rabbitmqadmin declare queue.
+
+    ./rabbitmqadmin declare queue --vhost=Some_Virtual_Host name=some_outgoing_queue durable=True auto_delete=False
+
+    CLI Example:
+
+    .. code-block:: bash
+
+        salt '*' rabbitmq declare_queue '<queue_name>' '<vhost_name>' True
+
+
+    Setup:
+    rabbitmqadmin is from http://localhost:15672/cli/rabbitmqadmin it needs to be in the path.
+    see rabbitmq-formula/rabbitmq/rabbit-management.sls
+
+    '''
+
+    durable = translate_boolean(durable)
+    auto_delete = translate_boolean(auto_delete)
+
+    if runas is None:
+        runas = salt.utils.get_user()
+    res = __salt__['cmd.run']('rabbitmqadmin declare queue --vhost={0} name={1} durable={2} auto_delete={3}'.format(
+        vhost, name, durable, auto_delete),
+                              python_shell=False, runas=runas)
+    log.debug(res)
```

Comment:
It's going to be extremely difficult to discern what these log messages are or where they are coming from. Please either change these to include info about their origin or take them out.